python/lib/core/dmod/core/meta_data.py

```diff
@@ -157,7 +206,7 @@ def time_series_index(self) -> Optional[str]:
 
 class ContinuousRestriction(Serializable):
```

Comment:
The intent was for it to not be for that exclusively; there is the `TimeRange` subtype for that purpose.  But the datetime format was moved to this level to help implement serialization and deserialization, so that you can call the deserialization function for `ContinuousRestriction` and get a `TimeRange` object if appropriate.